drafts/dec-14.md

```diff
@@ -0,0 +1,9 @@
+Here are some of the highlights for the [December 14](https://github.com/ipfs/pm/issues/76) Sprint:
+
+### Releases
+* @whyrusleeping shipped [fs-repo-migrations](https://github.com/ipfs/fs-repo-migrations)! These are migrations for the filesystem repository of ipfs clients. This tool is written in Go, and developed alongside go-ipfs, but it should work with any repo conforming to the fs-repo specs.
+* @Dignifiedquire did some great work with [rust-multiaddr](https://github.com/Dignifiedquire/rust-multiaddr), which is a Rust implementation of @jbenet's [multiaddr](https://github.com/jbenet/multiaddr).
+### Updates
+* @diasdavid worked on the npm on IPFS project, at [registry-mirror](https://github.com/diasdavid/registry-mirror). This involved some new features, moving the mirror to a different server, and making it work better with larger dirs and with 0.4.0. While doing all of this, he still found time to do some major updates for [js-ipfs](https://github.com/ipfs/js-ipfs-repo), too.
```

Comment:
It would be cool to add "action calls", for example, people can try registry-mirror today and give feedback (this would actually be really great), and we even created a tutorial + blog post :)